packages/reason-generate-types-from-graphql-schema/reason-generate-types-from-graphql-schema.0.7/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+maintainer: "greg <greg@hackages.io>"
+authors: "greg <greg@hackages.io>"
+build: [
+  ["ocamlbuild -r -use-ocamlfind  src/index.native"]
```

Comment:
this is a bit unconventional :-) What happens if the platform you are trying to build doesn't support native code? I would recommend you to have a look at [jbuilder](https://github.com/janestreet/jbuilder) which makes this kind of things easier.